### ISSUE TITLE
Not following original docs

### ISSUE THREAD

**Issue by nzw0301** [fchollet/keras-docs-ja#45]
as far as I know,
- [ ] optimizer ([Nadam](https://github.com/fchollet/keras/blob/master/keras/optimizers.py#L418-L436))
- [x] convolutional ([dim_ordering](https://github.com/fchollet/keras/commit/973ece98096c87871f72d17909f7c2883fa973c3#diff-d462a3275abee3aa77ba2b341e45cf20))
- [x] core ([merge](https://github.com/fchollet/keras/commit/91b930298ba30394b35a52830317c29989568199#diff-fe3ca734e8421a17729d0b0ea4c800ba))
- [x] normalization ([about mode=2](https://github.com/fchollet/keras/blob/master/keras/layers/normalization.py#L13-L28))
- [x] preprocessing/image
- [ ] convnet (atrousconvolution2d)
- [ ] backend

**Comment by amaotone** [comment]
additional tasks
- [x] utils
- [ ] metrics
- [x] callbacks

**Comment by nzw0301** [comment]
- [x] keras.json details

**Comment by nzw0301** [comment]
Since Keras 2 is released, many pages have not been following original docs.
This issue may confuse new translators, so I close this issue once.
